### ISSUE TITLE
False positive in  07-arrays problem

### ISSUE THREAD

**Issue by talves** [total-typescript/beginners-typescript-tutorial#27]
This could be construed as a tuple of type `Post`, but of course it's an array.

https://github.com/total-typescript/beginners-typescript-tutorial/blob/266c2f6ea1fcabd2778904e7c62ef96c13875942/src/07-arrays.problem.ts#L19-L24

It might be better to create an array example so it fails if a student set the user interface to `[Post]` which passes the type checks.

```js
  posts: [
    {
      id: 1,
      title: "How I eat so much cheese",
    },
    {
      id: 2,
      title: "Why I don't eat vegetables",
    },
  ],
```

**Comment by DongHY1** [comment]
#10

**Comment by mattpocock** [comment]
Makes sense - go for it!
